libcore/src/handlers/ecdsa.rs

```diff
@@ -47,7 +47,8 @@ use miden_processor::{AdviceMutation, EventError, EventHandler, ProcessState};
 use crate::handlers::{bytes_to_packed_u32_felts, read_memory_packed_u32};
 
 /// Qualified event name for the ECDSA signature verification event.
-pub const ECDSA_VERIFY_EVENT_NAME: EventName = EventName::new("stdlib::crypto::dsa::ecdsa::verify");
+pub const ECDSA_VERIFY_EVENT_NAME: EventName =
+    EventName::new("miden::core::crypto::dsa::ecdsa::verify");
```

Comment:
Related to one other comment: I think the event name here should be `miden::core::crypto::dsa::ecdsa_k256_keccak::verify`